connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -171,51 +181,44 @@ export async function confluenceListPageIdsInSpaceActivity({
 }
 
 async function upsertConfluencePageInDb(
-  connectionId: string,
-  dataSourceConfig: DataSourceConfig,
-  page: ConfluencePageWithBodyType
+  connectorId: ModelId,
+  page: ConfluencePageWithBodyType,
+  visitedAtMs: number
 ) {
-  const connector = await Connector.findOne({
-    where: {
-      connectionId,
-      dataSourceName: dataSourceConfig.dataSourceName,
-      workspaceId: dataSourceConfig.workspaceId,
-    },
-  });
-
-  if (!connector) {
-    throw new Error(`Connector not found (connectionId: ${connectionId})`);
-  }
-
   await ConfluencePage.upsert({
-    connectorId: connector.id,
+    connectorId,
     pageId: page.id,
     spaceId: page.spaceId,
     parentId: page.parentId,
     title: page.title,
     externalUrl: page._links.tinyui,
     version: page.version.number,
+    lastVisitedAt: new Date(visitedAtMs),
   });
 }
 
 interface ConfluenceUpsertPageActivityInput {
-  spaceId: string;
-  spaceName: string;
-  pageId: string;
-  dataSourceConfig: DataSourceConfig;
-  isBatchSync: boolean;
   connectionId: string;
   connectorId: ModelId;
+  dataSourceConfig: DataSourceConfig;
```

Comment:
I thiiiink we generally want to pull it back from the activity? (will also work on normalizing that across connectors)